smoke/IotEdgeQuickstart/Program.cs

```diff
@@ -84,6 +84,18 @@ class Program
         [Option("-h|--use-http=<hostname>", Description = "Modules talk to iotedged via tcp instead of unix domain socket")]
         public (bool useHttp, string hostname) UseHttp { get; } = (false, string.Empty);
 
+        [Option("--use-connect-management-uri=<connect_management_uri>", Description = "Modules talk to a custom connect management socket (default is unix:///var/run/iotedge/mgmt.sock)")]
```

Comment:
If this parameter is not provided, it will use the value in config.yaml.  Don't put unix:// as default here, what if we change that value in config.yaml in the future.

Reply:
This is just a description to the input parameters. I can remove the default socket values from the parameters descriptions if such clarification is unnecessary.